app/models/like.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Like < ApplicationRecord
+  belongs_to :likeable, polymorphic: true
+  has_one :user
```

Comment:
- [ ] each like should also belong to one post

Reply:
https://github.com/raheebwa/irla-facebook/blob/5b15c0d6476da84a7e7095a5647472b45c511428/app/models/like.rb#L3-L9

Solved!